bls/bls.go

```diff
@@ -442,7 +443,7 @@ func (sk *SecretKey) GetPublicKey() *PublicKey {
 	// https://github.com/miracl/core/blob/fda3416694d153f900b617d7bc42038df34a2da6/go/TestMPIN.go#L79
 	const MGS = BN254.MGS
 	const MFS = BN254.MFS
-	const G1S = 2*MFS + 1 /* Group 1 Size */
+	// const G1S = 2*MFS + 1 /* Group 1 Size */
```

Comment:
Btw why it was removed from here?

Reply:
We can't remove variables exported. Because it might be used by 3rd package. golang-lint can't detect it properly.